README.md

```diff
@@ -1,265 +1,85 @@
+CDI works with standard core Kubernetes resources and is storage device agnostic, while its primary focus is to build disk images for Kubevirt, it's also useful outside of a Kubevirt context to use for initializing your Kubernetes Volumes with data.
 
-- Patch: Mid development cycle critical bug fix. In the case that a Minor release has a bug that must be fixed for users before the next release cycle, a Patch may be published.  Patches should be small in scope and only alter / introduce code related to the bug fix.
 
-See [releases.md](/doc/releases.md) for more information on versioning.
+# Introduction
 
-### Data Format
+## Kubernetes extension to populate PVCs with VM images
+CDI provides the ability to populate PVCs with VM images upon creation.  Multiple image formats and sources are supported:
 
-The importer is capable of performing certain functions that streamline its use with Kubevirt.  It automatically decompresses **gzip** and **xz** files, and un-tar's **tar** archives. Also, **qcow2** images are converted into a raw image files needed by Kubevirt, resulting in the final file being a simple _.img_ file.
+### Current supported Image formats
+* .tar
+* .gz
+* .xz
+* .img
+* .iso
+* .qcow2
 
-Supported file formats are:
+### Current supported image endpoints
+* http
+* S3
+* local directory
 
-- .tar
-- .gz
-- .xz
-- .img
-- .iso
-- .qcow2
+## DataVolumes
+CDI also includes a CRD, that provides an object of type DataVolume.  The DataVolume is an abstraction on top of the standard Kubernetes PVC and can be used to automate creation and population of a PVC for consumption in a Kubevirt VM.
 
-## Deploying CDI
+## Deploy it
 
-### Assumptions
-- A running Kubernetes cluster with roles and role bindings implementing security necesary for the CDI controller to watch PVCs and pods across all namespaces.
-- A storage class and provisioner.
-- An HTTP or S3 file server hosting VM images
+Deploying the CDI controller is straightforward. In this document the _default_ namespace is used, but in a production setup a [protected namespace](#protecting-the-golden-image-namespace) that is inaccessible to regular users should be used instead.
 
-- An optional "golden" namespace acting as the image registry. The `default` namespace is fine for tire kicking.
+  ```
+  $ VERSION=<cdi version>
+  $ kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator.yaml
+  $ kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator-cr.yaml
+  ```
 
-### Download CDI
+If you're running OpenShift, make sure you enable privileged containers
```

Comment:
I don't think this is needed anymore due to the operator @mhenriks keep me honest.

Reply:
correcct